examples/step-10/doc/results.dox

```diff
@@ -27,22 +27,38 @@ uniformly refined grid (right).
 
 <div class="twocolumn" style="width: 80%">
   <div>
-    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q1.svg">
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q1.svg"
+         alt="Five-cell discretization of the disk."
+         width="400" height="400">
   </div>
   <div>
-    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_1_q1.svg">
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_1_q1.svg"
+         alt="20-cell discretization of the disk (i.e., five cells
+              refined once)."
+         width="400" height="400">
   </div>
   <div>
-    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q2.svg">
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q2.svg"
+         alt="Five-cell discretization of the disk with quadratic edges. The
+              boundary is nearly indistinguishable from the actual circle."
+         width="400" height="400"
+         >
   </div>
   <div>
-    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_1_q2.svg">
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_1_q2.svg"
+         alt="20-cell discretization with quadratic edges."
+         width="400" height="400">
   </div>
   <div>
-    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q3.svg">
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q3.svg"
+         alt="Five-cell discretization of the disk with cubic edges. The
+              boundary is nearly indistinguishable from the actual circle."
```

Comment:
does breaking the string like this look sensible?

Reply:
I do not see the line break when I view this locally.

Reply:
It's just text -- HTML ignores the amount of whitespace.